examples/website/plot/plot-layer/axes-layer.ts

```diff
@@ -169,13 +179,17 @@ export default class AxesLayer extends Layer {
         strokeColor: color
       };
 
-      modelsByName.grids.setInstanceCount(numInstances);
-      modelsByName.labels.setInstanceCount(numInstances);
+      modelsByName.grids.setInstanceCount(instanceCount);
+      modelsByName.labels.setInstanceCount(instanceCount);
 
-      modelsByName.grids.setUniforms(Object.assign({}, uniforms, baseUniforms)).draw();
-      modelsByName.labels
-        .setUniforms(Object.assign({}, uniforms, baseUniforms, labelTexture))
-        .draw();
+      modelsByName.grids.setUniforms(Object.assign({}, uniforms, baseUniforms));
+      modelsByName.labels.setBindings({labelTexture});
+      modelsByName.labels.setUniforms(
+        Object.assign({}, uniforms, baseUniforms, labelTextureUniforms)
+      );
+
+      modelsByName.grids.draw(this.context.renderPass as $TODO);
+      modelsByName.labels.draw(this.context.renderPass as $TODO);
```

Comment:
`renderPass.id` is increasing each time we draw — is that expected, or could it indicate I've done something incorrectly here?

Reply:
This is happening elsewhere, e.g. in `ScatterplotLayer` and I think it is by design. Could you switch to the `8.9-release` branch and check the behavior there?

Reply:
It doesn't look like `this.context.renderPass` exists in 8.9, not sure if there's some comparable thing to check for? 

Checking the ArcLayer example on `master` branch — the render pass ID increments in the same way. So, if this is an issue at all, it doesn't appear to be specific to this PR.